addons/parachute/functions/fnc_cutParachute.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+	Name: Cut Parachute
+
+	Author: joko // Jonas
+
+	Description:
+	Cut Parachute and delete Old
+
+	Parameters:
+	0: Object
+
+	Returns:
+	Nothing
+*/
+private["_unit","_vehicle"];
+_unit = _this select 0;
+_vehicle = vehicle _unit;
+_unit action ["GetOut", vehicle _unit];
+deleteVehicle _vehicle;
+_unit addBackpack "B_Parachute";
```

Comment:
There are more than one type of parachutes in the game through mods (ACE adds the `ACE_NonSteerableParachute` variant besides `B_Parachute`).

The parachute added should be of the same class of that the unit originally had, which we probably need to store.